core/spec.md

```diff
@@ -2446,25 +2511,22 @@ Where:
 - a request to update a read-only attribute MUST be silently ignored
 - a request to update a mutable attribute with an invalid value MUST
   generate an error (this includes deleting a mandatory mutable attribute)
-- complex attributes that have nested values (eg. `tags`) MUST be specified
+- complex attributes that have nested values (eg. `labels`) MUST be specified
   in their entirety
 
-Missing Registry HTTP headers MUST be interpreted as a request to delete the
-attribute.
-
-Note: `latestId` is a read-only attribute in this context and therefore
-if present MUST be silently ignored.
+Missing Registry HTTP headers MUST be interpreted as a request to leave the
+corresponding Version attribute unchanged. An attribute with a value of `null`
```

Comment:
```suggestion
corresponding attribute unchanged in the new Version. An attribute with a value of `null`
```

(Otherwise it sounds like it's about an attribute called Version.)